wqflask/wqflask/static/new/css/markdown.css

```diff
@@ -11,3 +14,54 @@
     margin-right: auto;
     margin-left: auto;
 }
+
+
+
+.github-btn {
+    margin: 20px 10px;
+    display: flex;
+    width: 90vw;
+    justify-content: flex-end;
+}
+
+.github-btn a {
+    display: inline-block;
+    align-self: center;
+    text-decoration: none;
+    transition-property: all;
+    transition-duration: 0.5s;
+    padding: 8px 12px;
+    color: #fff;
+    /*background: #27407a;*/
```

Comment:
**issue:nitpick:** Please remove commented out code

Commented out codes, most times, make code abit confusing to read. Let's get in to the habit of removing commented out blocks from the final commits...